content/go/concepts/math-functions/terms/tan/tan.md

```diff
@@ -23,8 +23,8 @@ result := math.Tan(angle)
 
 Where `result` is the tangent value of `angle`, returned as a float, except under the following circumstances:
 
-- The result of `Tan(-Inf)` is `NaN`
-- The result of `Tan(+Inf)` is `NaN`
+- The result of `Tan(math.Inf(+1))` is `NaN`
+- The result of `Tan(math.Inf(-1))` is `NaN`
```

Comment:
I don't this should be changed. I don't really see a problem with the shorthand in the syntax section, I believe almost all of our entries use this format. Additionally, if we were to follow through with this approach then we would need to change the last line to Tan(math.NanN()). I would suggest reverting these changes

Reply:
Sure, no problem! I apologize for this… just found the open _issue_ listed when I was looking for an available _'first issue'_ to solve, as part of Codecademy's **"Learn Git & GitHub"** course.
Also, thinking in a positive way on this mistake, that's a good opportunity to ask what comes next… should I terminate this PR? Or the team will do this, and close the corresponding _issue_?